.env

```diff
@@ -1,3 +1,6 @@
 VITE_APP_GRAPHQL_URI_EXCHANGE=https://graph.ipfs1.dev.infra.soramitsu.co.jp/subgraphs/name/klaytn-subgraph/exchange
 VITE_APP_GRAPHQL_URI_FARMING=https://graph.ipfs1.dev.infra.soramitsu.co.jp/subgraphs/name/klaytn-subgraph/farming
 VITE_APP_GRAPHQL_URI_STAKING=https://graph.ipfs1.dev.infra.soramitsu.co.jp/subgraphs/name/klaytn-subgraph/staking
+VITE_APP_GRAPHQL_URI_SNAPSHOT=https://hub.snapshot.org/graphql
+
+VITE_APP_SNAPSHOT_SPACE=uniswap
```

Comment:
Should be defined at `env.d.ts` as well.

Reply:
Done